source/develop/release-management/features/virt/enhance-import-export-with-ova.md

```diff
@@ -0,0 +1,139 @@
+better compatibility with other platforms.  
+
+The OVA file would be created at a given path within a particular host (this capability is targeted for extracting the VM/template into an external storage used for backup/migration). In addition, it will include neither snapshots nor the template that the VM may be based on.  
+
+## Enhance Import OVA
+This feature enhances the current support provided for import OVA files in two ways. First, it would be possible to specify a path to OVA that was generated by oVirt. Import of OVA that was generated by oVirt does not involve conversion and thus, should be faster than importing OVA files generated by others.  
+
+Second, in order to provide better support for typical backup/restore and migration processes we enable listing all the OVA files in a given directory.  
+
+## Download OVA
+This feature will leverage the implementation of the planned download-image to provide similar functionality for OVA files.  
+
+# Design
+This section details the high-level design of the different aspects described in the previous section.
+
+## Upload OVA
+In principle, uploading an OVA is similar to uploading an image. However, it is more challenging because OVA is a more complex resource than an image.  
+
+The upload process would be implemented differently depending on whether the OVA was generated by oVirt or by other source (and thus requires conversion). In both cases, the user will be able to select a file from within a browser and be presented with the VM configuration that will allow him to configure the parameters for the import process:
+
+1. The user selects an OVA file from within the browser.
+2. The front-end sends the OVF to the back-end.
+3. The back-end parses the OVF and returns back an object that represents the VM.
+4. The front-end displays the VM configuration and allows the user to configure the parameters for the import (namely, VM name, destination storage, and networks).
+
+From this point on, the process depends on the source of the OVA. Note that the source is returned to the front-end by the back-end in the _origin_ field of the VM.  
+
+### Upload OVA generated by oVirt
+If the OVA was generated by oVirt, the rest of the process will be:
+
+1. The back-end adds the VM entity into the database.
+2. The back-end creates the destination volumes for the VM disks using the existing process for this in oVirt.
+3. The back-end requests the imageio proxy to prepare to upload-ova with a list of parameters per disk. Namely, these parameters contain the destination path of the disk and its actual size.
+4. The proxy returns a URL (similar to the upload-image flow).
+5. The back-end passes this URL to the front-end (similar to the upload-image flow).
+6. The front-end starts transmitting the **disks** one by one.
```

Comment:
who does the separation of one OVA file into the individual disks? FE? imagio-proxy?